src/schema/nmdc.yaml

```diff
@@ -350,11 +378,16 @@ classes:
   omics processing:
     aliases:
       - omics assay
+      - sequencing project
+      - experiment
     is_a: biosample processing
     in_subset: 
       - sample subset
     description: >-
       The methods and processes used to generate omics data from a biosample or organism.
+    alt_descriptions:
+      embl.ena: >-
+        An experiment contains information about a sequencing experiment including library and instrument details.
```

Comment:
Do you want to say an experiment "produces" information? The "contains" information sounds odd to me.

Reply:
That's not my definition. See https://w3id.org/linkml/AltDescription - this is a description attributed to EMBL.ENA. See https://ena-docs.readthedocs.io/en/latest/submit/general-guide/metadata.html

Reply:
I wasn't aware of the ENA descriptions. What the value gained by adding the enum instead just the alternative description? If you are wanting to match fields across schemas, wouldn't it be better to use on of the mapping slots?